library/core/src/iter/traits/collect.rs

```diff
@@ -358,3 +358,61 @@ impl Extend<()> for () {
     }
     fn extend_one(&mut self, _item: ()) {}
 }
+
+#[stable(feature = "extend_for_tuple", since = "1.55.0")]
```

Comment:
1.54 was released yesterday, so nightly is now 1.56

```suggestion
#[stable(feature = "extend_for_tuple", since = "1.56.0")]
```